core/src/main/resources/hudson/AboutJenkins/index.jelly

```diff
@@ -22,57 +22,143 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 -->
 
-<!-- 3rd party license acknowledgements and  -->
+<!-- About Jenkins page -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:s="/lib/form">
+<j:jelly xmlns:j="jelly:core" xmlns:l="/lib/layout" xmlns:t="/lib/hudson">
   <l:layout permissions="${app.MANAGE_AND_SYSTEM_READ}" title="${%about(app.VERSION)}">
-    <st:include page="sidepanel.jelly" />
-    <l:main-panel>
-      <div class="jenkins-app-bar">
-        <div class="jenkins-app-bar__content">
-          <h1>
-            ${%about(app.VERSION)}
+    <l:side-panel>
+      <style>
```

Comment:
can't you move this to the less file?

Reply:
Ah it's just so this page is styled, it'll be removed in a future PR when I get around to looking at some side panel updates :)